### ISSUE TITLE
saving relative path for plugin files problem

### ISSUE THREAD

**Issue by Houston4444** [falkTX/Carla#1097]
Hi.
I was wondering why sometimes Carla doesn't achieve to load the convolution .wav file with rgareus LV2 Convolution (Mono, Stereo, or Mono -> Stereo, doesn't matters), when I load the carla file (with RaySession, but I don't think it changes anything here).

in the .carxp file, I saw that this path is a relative path.
The problem is that this relative path is relative to the current PWD, I think it would be more relevant to be relative to the .carxp file.

Don't you think ?

**Comment by falkTX** [comment]
I agree. need to handle this better. likely not in v2.2, but in whatever comes after.
